uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/entitlement-data.js

```diff
@@ -0,0 +1,251 @@
+define(function () {
+  return {
+    all: function (params, withBalance) {
```

Comment:
I don't like this tbh. The mocked data should be just that, raw data. Why do you need to add methods (`all`, `breakdown`, `totalEntitlements`) to it? The mocked api service should be in charge of manipulating it